.github/workflows/ci.yml

```
@@ -661,23 +661,22 @@ jobs:
     needs: [test-builds-arch, test-builds-os, build-sdist, check]
     if: always() && startsWith(github.ref, 'refs/tags/')
     runs-on: ubuntu-latest
-    environment: release
+
+    environment:
+      name: release
 
     permissions:
       id-token: write
 
     steps:
       - uses: actions/checkout@v4
 
-      - name: set up python
-        uses: actions/setup-python@v5
-        with:
-          python-version: '3.13'
+      - uses: astral-sh/setup-uv@v6
 
       - run: pip install -U twine
```

```suggestion
      - run: uv pip install -U twine
```

Maybe? I'm not sure pip will be available in this context

---

good point